src/test/java/org/apache/cassandra/sidecar/HealthServiceIntegrationTest.java

```diff
@@ -0,0 +1,259 @@
+/*
```

Comment:
nit: Should place all integration tests under `src/integration`? or different package inside `test` itself